### PR TITLE
Point DC/OS 1.13-dev to Mesos master.

### PR DESCRIPTION
**Pull request by rukletsov**
## High-level description

This change switches DC/OS `master` (= `1.13-dev`) to Mesos `master` (= `1.8-dev`).

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

## Related tickets (optional)

Other tickets related to this change:

## Checklist for all PRs

  - [ ] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by rukletsov** [comment]
@mesosphere-mergebot bump-ee

File: packages/mesos-modules/buildinfo.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "requires": [
-    "mesos", 
+    "mesos",
```
Comment (bbannier) [inline packages/mesos-modules/buildinfo.json]: To remove noise from this patch and any automated patch building on top of it, I'd suggest to leave the EOL spaces we unfortunately get from our tooling.

Here and below.

**Comment by rukletsov** [comment]
@mesosphere-mergebot override-status teamcity/dcos/build/dcos/windows https://jira.mesosphere.com/browse/DCOS-41386

**Comment by bbannier** [comment]
@mesosphere-mergebot override-status teamcity/dcos/build/dcos/windows https://jira.mesosphere.com/browse/DCOS-41386

File: packages/mesos-modules/buildinfo.json (hunk 1 above)
Comment (rukletsov) [reply packages/mesos-modules/buildinfo.json]: https://github.com/mesosphere/mesos-ci/issues/36

**Comment by rukletsov** [comment]
@mesosphere-mergebot label Ready For Review

**Pull request merged**